src/_tutorials/dart-vm/httpserver.md

```diff
@@ -527,24 +527,24 @@ Run the server and client on the command line.
 
 1. First the server:
```

Comment:
Let's change this to:

First, run the server:

Reply:
done